NonsoAmadi10/Auto-Mart
# #166788643 Creates View Unsold Cars feature

### What does this PR do??
- Creates ` GET /api/v1/car?status=available `

### Description of Task Taken To Complete??
- Setup TestCases
- Implement feature by persisting to a database


### How should this be manually Tested??
- Clone this Repo
- ` git checkout ft-user-should-view-unsold-car-166788643 `
- Run ` npm Install ` to Install dependencies
- Setup environment variables in the .env.example file
- Start the server by running ` npm run dev-start `
- Open postman on ` GET /api/v1/car?status=available `

### Any Background Context you want to provide ??
- You must pass in a token in the request header. see #50 and #51 

### What are the relevant pivotal tracker stories??
- [166788643](https://www.pivotaltracker.com/story/show/166788643)
